src/test/java/org/arquillian/container/chameleon/ManualSettingsXmlTestCase.java

```diff
@@ -0,0 +1,95 @@
+package org.arquillian.container.chameleon;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.RunWith;
+import org.junit.runner.notification.Failure;
+import org.junit.runners.model.InitializationError;
+
+/**
+ * @author <a href="mailto:mjobanek@redhat.com">Matous Jobanek</a>
+ */
+public class ManualSettingsXmlTestCase {
```

Comment:
Interesting tests :) I would consider two things here

- mark it as not thread safe, because if we run them in parallel we can have race conditions on system properties. You can use this annotation: http://jcip.net/annotations/doc/net/jcip/annotations/NotThreadSafe.html
- for files you can have a look at JUnit rule: http://junit.org/junit4/javadoc/4.12/org/junit/rules/TemporaryFolder.html